eta-meeting-organizer-frontend/src/app/meeting-room/components/meeting-room.component.ts

```diff
@@ -1,77 +1,58 @@
 import { Component } from '@angular/core';
-import { FormControl, FormGroup } from '@angular/forms';
-import { TranslateService } from '@ngx-translate/core';
+import { Observable } from 'rxjs';
+import { ApiCommunicationService } from '~/app/shared/services/api-communication.service';
+import { MeetingRoom } from './../../models/meetingroom.model';
 
 @Component({
-  selector: 'app-welcome-component',
+  selector: 'app-meetingRoom',
   styles: [`
-    .card{
-      margin-top: 10%;
-      width: 50%;
-    }
     .row {
       min-height: calc(100vh - 60px);
     }
-    .field{
-      width:100%;
-    }
-    .toggle{
-      margin-top:5%
+    table {
+      width: 100%;
     }
   `],
   template: `
-   <div class="row justify-content-center align-items-center">
-    <div class="col-5">
-<form [formGroup]="profileForm">
-  <mat-card class="card">
-    <p>
-      <mat-form-field class="field">
-        <mat-label>{{'meeting-room.text' | translate}}</mat-label>
-        <input type="text" formControlName="name" matInput placeholder="{{'meeting-room.placeholderName' | translate}}">
-      </mat-form-field>
-    </p>
-      <mat-form-field class="field">
-        <mat-label>{{'meeting-room.selectBuilding' | translate}}</mat-label>
-        <mat-select formControlName="building">
-          <mat-option value="option1">{{'meeting-room.building' | translate}} </mat-option>
-          <mat-option value="option2">{{'meeting-room.building1' | translate}}</mat-option>
-        </mat-select>
-      </mat-form-field>
-    <br>
-      <mat-slide-toggle class="toggle" formControlName="projector">{{'meeting-room.projector' | translate}}
-      </mat-slide-toggle>
-    <br>
-    <p>
-      <mat-form-field class="field">
-        <mat-label>{{'meeting-room.seats' | translate}}</mat-label>
-        <input  type="number" formControlName="seats"
-          matInput placeholder="{{'meeting-room.placeholderSeats' | translate}}">
-      </mat-form-field>
-    </p>
-    <button mat-raised-button color="warn">{{'meeting-room.saveButton' | translate}}</button>
-  </mat-card>
-</form>
-</div>
-</div>
-`
+    <div class="row justify-content-center">
+      <table mat-table [dataSource]="meetingRoom$" class="mat-elevation-z8">
```

Comment:
megjelenik bármi is async pipe nélkül?

Reply:
megjelent minden adat amit kértem viszont most beraktam az async-ot